includes/class-sensei-modules.php

```diff
@@ -1566,6 +1580,17 @@ public function course_modules_title() {
 	 * @return void
 	 */
 	public function load_course_module_content_template() {
+		/**
+		 * Access check for the course lessons.
+		 *
+		 * @since 2.2.0
+		 *
+		 * @param bool $show_lessons Whether or not the lessons should be shown. Default true.
+		 * @param int  $course_id    Course ID.
+		 */
+		if ( ! apply_filters( 'sensei_course_lessons_has_access', true, get_the_ID() ) ) {
```

Comment:
See above comment re: copying the filter and documentation.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2763/commits/14e2809502dacf0478bdb18585a221f60c7806aa.